src/gui/HelpDialog.cpp

```diff
@@ -504,6 +504,19 @@ void HelpDialog::updateAboutText(void) const
 		     "Atque", "Worachate Boonplod"});
 	contributors.sort();
 
+	typedef QPair<QString, int> donator;
+	QVector<donator> financialContributors = {
+		{ "Laurence Holt", 1000 }, { "John Bellora", 470 }, { "Vernon Hermsen", 324 },
```

Comment:
I am not sure the actual numbers should be listed. People may not even want that (we did not ask them before!). We can list the people e.g. alphabetically or sorted by amount.